src/lib/components/HttpErrors/HttpErrors.js

```diff
@@ -60,24 +60,27 @@ ForbiddenComponent.defaultProps = defaultProps;
 
 export const Forbidden = Overridable.component('Forbidden', ForbiddenComponent);
 
-function NotFoundComponent(props) {
+function HttpErrorComponent(props) {
```

Comment:
If you want to avoid the renaming, the fix of the imports, and the other PR (pseudocode):
```javascript
function HttpErrorComponent(props) {
...
}


function NotFoundComponent(props) {
  return HttpErrorComponent('Page Not Found', ...)

```

basically, simply create the generic HttpErrorComponent, require title, message, optionally icon. Then, use it in the specialized `NotFoundComponent` passing the right props.